src/pages/audio/index.vue

```diff
@@ -1,11 +0,0 @@
-<template><VFourOhFour /></template>
-
-<script>
-import { defineComponent } from '@vue/composition-api'
-import VFourOhFour from '~/components/VFourOhFour.vue'
-
-export default defineComponent({
-  components: { VFourOhFour },
-  layout: 'blank',
-})
-</script>
```

Comment:
I _knew_ there had to be a better way to do this. Thank you for removing the need for these!